### PR TITLE
Convert settingsView to Tailwind

### PR DESCRIPTION
**Pull request by OceanOak**
Changelog:

```
Internal
- Update settingsView to use TailwindCSS
```

**Changes:**
- Convert settingsView to Tailwind.
- Remove unused CSS.

**Before -> After**

Frame 12 (4)

### CODE REVIEW

File: client/src/components/Settings/SettingsView.res [hunk 1]
```diff
@@ -39,7 +39,10 @@ let tabTitleView = (tab: T.Tab.t): Html.html<msg> => {
 
     Html.h3(
       list{
-        Attrs.classList(list{("tab-title", true), ("selected", isSameTab)}),
+        Attrs.classList(list{
+          (%twc("h-full m-0 mb-[-3px] px-2.5 py-0 text-grey2 cursor-pointer"), true),
```
Comment (pbiggar) [inline client/src/components/Settings/SettingsView.res:43]: I'd rather not use the `[]` syntax, especially with pixels, as tailwind AFAIK uses `rem`s which support zoom better.

To remove them, we can either add new presets, or if it looks the same either way (eg often 1 pixel doesn't matter), just round them to the nearest existing preset.

File: client/src/components/Settings/SettingsView.res [hunk 2]
```diff
@@ -50,13 +53,22 @@ let tabTitleView = (tab: T.Tab.t): Html.html<msg> => {
...
 }
 
 let settingViewWrapper = (acc: T.t): Html.html<msg> => {
   let tabView = settingsTabToHtml(acc)
   Html.div(
-    list{Attrs.class("settings-tab-wrapper")},
+    list{
+      Attrs.class(
+        %twc(
+          "w-[calc(100%-30px)] flex flex-col h-full overflow-auto mt-2.5 mr-[5px] mb-2.5 ml-[35px] scrollbar-none"
```
Comment (pbiggar) [inline client/src/components/Settings/SettingsView.res:68]: Can you change 30px into rems?

Comment (pbiggar) [reply client/src/components/Settings/SettingsView.res:68]: `mt` and `mb` are the same so could be combined into `my`?

File: client/src/components/Settings/SettingsView.res [hunk 3]
```diff
@@ -84,7 +100,11 @@ let html = (m: AppTypes.model): Html.html<msg> => {
 
   Html.div(
     list{
-      Attrs.class("settings modal-overlay"),
+      Attrs.class(
+        %twc(
+          "z-[100] fixed top-0 left-0 w-full h-full bg-[#484848cc] flex items-center justify-center"
```
Comment (pbiggar) [inline client/src/components/Settings/SettingsView.res:105]: I think there's a feature to allow us do opacity directly: https://tailwindcss.com/docs/background-color#changing-the-opacity

So we could use `bg-grey1/80` I think?

File: client/src/components/Settings/SettingsView.res (hunk 1 above)
Comment (pbiggar) [reply client/src/components/Settings/SettingsView.res:43]: (this comment applies to anywhere that `[]` is used)

**Review by pbiggar — CHANGES_REQUESTED** [review]
Looks great, thanks!

**Comment by OceanOak** [comment]
This view broke while converting settingsView to Tailwind. Trying to figure out why.

image

**Pull request merged**